src/styles/components/_product-content.scss

```diff
@@ -1,43 +1,75 @@
 .product-content {
   display: grid;
   grid-template-columns: 1fr;
-  grid-template-rows: rem(300px) repeat(2, 1fr);
+  grid-template-rows: rem(500px) repeat(2, 1fr);
   grid-template-areas:
     'media'
     'detail'
     'information';
 
+  @include respond-above('sm') {
+    grid-template-rows: rem(620px) repeat(2, 1fr);
+  }
+
   @include respond-above('md') {
     grid-template-columns: repeat(2, 1fr);
-    grid-template-rows: repeat(2, 1fr);
+    grid-template-rows: rem(520px) 1fr;
     grid-template-areas:
       'media detail'
       'information information';
   }
 
-  & > .product-content__media,
+  @include respond-above('lg') {
+    grid-template-rows: rem(670px) 1fr;
+  }
+
+  @include respond-above('xl') {
+    grid-template-rows: rem(780px) 1fr;
+  }
+
   & > .product-content__detail,
   & > .product-content__information {
-    border-radius: rem(9px);
     margin-bottom: rem(16px);
 
+    border-radius: rem(9px);
+
     background-color: $light;
 
-    // TODO: Modify these when making other sections in Product View screen
     @include respond-above('md') {
+      margin-bottom: rem(24px);
+
       border-radius: unset;
-      margin-bottom: rem(32px);
     }
 
     @include respond-above('lg') {
-      margin-bottom: rem(48px);
+      margin-bottom: rem(30px);
     }
   }
 
+  & > .product-content__media,
+  & > .product-content__detail {
+    margin-top: rem(16px);
+  }
```

Comment:
```suggestion
  &__media,
  &__detail {
    margin-top: rem(16px);
  }
```

Reply:
from the original file we use `& > .product-content__media` and to change it to be `&__media` is it change anything? Just asking so I can implement it in further tasks. Thanks.

Reply:
It will change nothing because the ``&`` is a nest styles for the element and modifier within the same parent declaration block which this one the ``&`` has a meaning of its class name ``.product-content`` already.